model/log.go

```diff
@@ -120,8 +124,10 @@ func (item *LogLineItem) OneSecondNewer(previousItem interface{}) bool {
 
 // DownloadLogLines returns log lines for a given build ID and test ID. If the
 // test ID is empty, this will return all logs lines in the build.
-func DownloadLogLines(ctx context.Context, buildID string, testID string) (chan *LogLineItem, error) {
-	buildKeys, err := getBuildKeys(ctx, buildID)
+func DownloadLogLines(ctx context.Context, tracer otelTrace.Tracer, buildID string, testID string) (chan *LogLineItem, error) {
+	_, span := tracer.Start(ctx, "DownloadLogLines")
```

Comment:
If we allow the returned ctx to override ctx it will allow the trace that continues with `getBuildKeys` to be threaded together.